src/components/Footer/index.js

```diff
@@ -37,62 +39,109 @@ export const OfficeStyled = styled(Row)`
   overflow: hidden;
 `
 
-const Footer = () => {
-  return (
-    <GreyFooter>
-      <Padding top={{ smallPhone: 3, tablet: 4 }}>
+const DesktopGDPR = styled.div`
+  visibility: hidden;
+  position: absolute;
+  right: 0;
+  color: white;
+
+  a {
+    margin-left: 24px;
+  }
+
+  ${breakpoint('desktop')`
+    visibility: visible;
+  `}
+`
+
+const MobileGDPR = styled.div`
+  color: white;
+
+  a {
+    display: inline-block;
+    margin-right: 24px;
+    margin-bottom: 24px;
+  }
+
+  ${breakpoint('desktop')`
+    visibility: hidden;
+  `}
+`
+
+const LinkUnderline = styled(Link)`
+  text-decoration: underline;
+`
+
+const gdprLinksElement = gdpr.map(link => (
```

Comment:
It's probably more declarative/readable to have this map sitting inside the JSX below. We don't need to extract it out here, since we're not re-using it anywhere.